### PR TITLE
Always get country code by calling function which checks array boundary overflow

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by autarch** [comment]
Closes #53 (at least in part).

File: libGeoIP/GeoIP.c [hunk 1]
```diff
@@ -1615,7 +1615,7 @@ const char *GeoIP_country_code_by_name_v6_gl(GeoIP * gi, const char *name,
 {
     int country_id;
     country_id = GeoIP_id_by_name_v6_gl(gi, name, gl);
-    return (country_id > 0) ? GeoIP_country_code[country_id] : NULL;
+    return (country_id > 0) ? GeoIP_code_by_id(country_id) : NULL;
```
Comment (oschwald) [inline libGeoIP/GeoIP.c]: I think this can be just:

```c
return GeoIP_code_by_id(country_id);
```

(and the same for all the others).

Comment (oschwald) [inline libGeoIP/GeoIP.c]: Ah, never mind. I guess this excludes `0`

Comment (autarch) [inline libGeoIP/GeoIP.c]: Yeah `0` is a very special number here. I tried it the sane way originally but that caused test failures.

**Pull request merged**